### ISSUE TITLE
Recovery tool ISO creation

### ISSUE THREAD

**Issue by LiohMoeller** [advancingu/XPS13Linux#16]
I have updated to the latest version of Dell Recovery. I had never created a recovery image before and selected 'Only Create ISO' in the menu. The application closes directly afterwards and only an ISO with 0Byte is created in my Downloads folder.

**Comment by advancingu** [comment]
Is this not the same issue as #7?

**Comment by LiohMoeller** [comment]
No, this occurs after installation, using the latest version from the repo.

**Comment by tarballz** [comment]
I found a Dell OEM Ubuntu iso if it would make your life easier.  For some reason, you must boot it as a DVD, not on a flash drive.
https://drive.google.com/file/d/0BxuBIWrIIPFwTjVBZWxIWmx4VGs/view?pli=1

**Comment by hg8** [comment]
@tarballz Are you sure this is the correct iso for the XPS 13 ? The file name is dell-bto-trusty-**m3800-A08**-iso-20150116-0.iso

**Comment by tarballz** [comment]
Well I got two answers. I used this exact image to flash my xps13, because I had to restore the uefi partition. And it worked no problem. 
Thing is, I was pretty salty towards Dell after this shitshow of a distro (which also has an outdated kernel). So once everything was up and running, I flashed the / mount point with Elementary OS, and everything works great with it. Jumping from 3.13 to 3.19 was night and day. So many more things worked out of the box. 
I just started posting on the Dell forums too, since there seems to be a lot of people stranded.

**Comment by jared-dominguez** [comment]
@tarballz We use the LTS kernel used by Ubuntu 14.04 (which is an LTS release). Some people prefer running a newer kernel. Not everyone (especially our business customers) does.

**Comment by tarballz** [comment]
@jared-dominguez no it's not, check it out
https://wiki.ubuntu.com/TrustyTahr/ReleaseNotes#Kernel

It's 3.19.

I don't know what it is then man, on my FIRST BOOT with the Dell OEM version I got an Internal Error, and nonstop bugs.  Since I ditched it with a well-maintained distro, little to no errors.  The only errors I'm having right now are just annoyances.

I bought this laptop to develop other things that aren't bugs with Dell's distro.  I'm honestly blown away by the lack of initiative on Dell's part dealing with this.

**Comment by jared-dominguez** [comment]
@tarballz 3.19 is the Vivid hardware enablement (HWE) kernel only available as the default when using the Ubuntu 14.04.3 point release installation media. (It's in the Ubuntu 14.04 apt repository as something you can install, but that takes a manual effort.) That kernel has a 12 month support life: https://wiki.ubuntu.com/Kernel/LTSEnablementStack
In contrast, the 3.13 kernel in Ubuntu 14.04 is supported for two years and nine months after the Vivid HWE kernel is no longer supported. We only use the HWE kernels when absolutely necessary, i.e. when needed for hardware enablement.

If you have specific bugs you encountered with our factory image (which itself is a stock Ubuntu 14.04 install with Canonical-supported backports for HWE), I encourage you to report them to Dell ProSupport.

**Comment by tarballz** [comment]
@jared-dominguez even if that is the case, the distro is inherently flawed. Have you been to the Dell forum? There's a huge number of people that are experiencing crippling bugs. Which led to the creation of this GitHub. There are so many bugs that there was an entire community created to deal with these bugs. I've been on the forum for the past couple days helping people with the exact same bugs that I've had. You obviously haven't even glanced at the forum. People who paid for support are getting shrugged off. This is unacceptable and the majority of the community feels the same.

**Comment by jared-dominguez** [comment]
@tarballz Yes, I have been to the TechCenter Project Sputnik forum. I work for Dell, am one of the most active people there and have been since it was created 3.5 years ago. Note that TechCenter is a community-driven forum. The support you pay for _must_ go through our support contacts, i.e. if you bought Dell ProSupport, to utilize that support contract you must contact Dell ProSupport. I (and occasionally others at Dell) monitor the TechCenter Sputnik forum and try to arrange callbacks (through our social media team) for customers for their convenience, but that is not a reliable method if you are needing to utilize your support contract.

I've been involved on the side with Project Sputnik since its inception and of a couple weeks ago became the development manager for Project Sputnik systems. My current focus is managing the development of the Skylake systems and on wrangling the current open issues for sustaining systems.

Flaws in Ubuntu 14.04 or not, that is what we use as it is the best common base for us to use. It is also what our business customers use and is more sustainably maintainable than other distributions and distribution versions with support lives that are shorter.

Yes, there have been issues with the 2015 XPS 13, including a delayed launch and a relaunch with a new factory image months later. We at Dell are aware of these and are working on several changes to improve the customer experience going forward.

FYI, this GitHub community was actually created because of the delayed launnch of 9343, i.e. it was created before we launched the 2015 XPS 13 Developer Edition.

For what it's worth, we have shipped _several thousand_ 2015 XPS 13 Developer Edition systems since this Ubuntu version of the 2015 XPS 13 became available in April of this year. Over the life of Project Sputnik, the program has resulted in a significant number of Developer Edition systems shipping worldwide and has significantly increased the profile of Linux on client systems within Dell. I say this not to discount those customers experiencing crippling bugs but to point out that there are also customers both not reporting issues and reporting not having any major issues.

Of those issues that are considered crippling, yes, we're aware of many of them and working to resolve them. For this issue in particular, it has long been resolved, including in the current version of our factory image. When you first replied to this issue, you were replying to a message from May.
